betterconf/exceptions.py

```diff
@@ -0,0 +1,17 @@
+class BetterconfError(Exception):
+    pass
+
+
+class ImpossibleToCastError(BetterconfError):
+    def __init__(self, val: str, caster: object):
+        self.val = val
+        self.caster = caster
+        self.message = 'Could not cast "{}" with {}'.format(val, caster.__class__.__name__)
```

Comment:
`pyproject.toml` says we're using Python 3.6+, why not use f-strings?

Reply:
Good catch.